diff --git a/japanese/helpers/file_ops.py b/japanese/helpers/file_ops.py
index a6a5898..5d727fc 100644
--- a/japanese/helpers/file_ops.py
+++ b/japanese/helpers/file_ops.py
@@ -52,9 +52,12 @@ def find_config_json() -> pathlib.Path:
 @functools.cache
 def user_files_dir() -> pathlib.Path:
     """Return path to the user files directory."""
-    for parent_dir in walk_parents(__file__):
-        if (dir_path := parent_dir.joinpath("user_files")).is_dir():
-            return dir_path
+    if pathlib.Path(os.getenv("ANKI_JAPANESE_DIR")).is_dir():
+        return pathlib.Path(os.getenv("ANKI_JAPANESE_DIR"))
+    else:
+        for parent_dir in walk_parents(__file__):
+            if (dir_path := parent_dir.joinpath("user_files")).is_dir():
+                return dir_path
     raise RuntimeError("couldn't find user_files directory")
 
 
